vision/module/region_of_interest.py

```diff
@@ -27,9 +24,15 @@ def region_of_interest(depth_frame, depth_val, center):
     Parameters
     ----------
     depth_frame: np array
```

Comment:
Same thing here, could also be defined as an ndarray for consistency (though not actually a functional issue).